src/hooks/useContributions.ts

```diff
@@ -35,51 +38,82 @@ const mockData: ContributionUser = {
   ],
 };
 
-export function useContributionUsers(): ContributionUser[] {
+export function useContributionUsers(): UserContributions {
   const integrations = useCurrentCircleIntegrations();
   const epoch = useSelectedCircle().circleEpochsStatus.currentEpoch;
-
   const responses = useQueries(
     epoch && integrations.data
       ? integrations.data
-          .filter(i => i.type === 'dework')
-          .map(i => ({
-            queryKey: `circle-integration-contributions-${i.id}-${epoch.id}`,
-            queryFn: () =>
-              fetch(
-                `https://api.deworkxyz.com/integrations/coordinape/${
-                  i.data.organizationId
-                }?epoch_start=${epoch.start_date}&epoch_end=${
-                  epoch.end_date
-                }&workspace_ids=${encodeURIComponent(
-                  i.data.workspaceIds?.join(',') || ''
-                )}`
-              )
-                .then(res => res.json())
-                .then(res => res as Response),
+          .filter(
+            integration =>
+              integration.type === 'dework' || integration.type === 'wonder'
+          )
+          .map(integration => ({
+            queryKey: `circle-integration-contributions-${integration.id}-${epoch.id}`,
+            queryFn: () => {
+              if (integration.type === 'dework') {
+                return fetch(
+                  `https://api.deworkxyz.com/integrations/coordinape/${
+                    integration.data.organizationId
+                  }?epoch_start=${epoch.start_date}&epoch_end=${
+                    epoch.end_date
+                  }&workspace_ids=${encodeURIComponent(
+                    integration.data.workspaceIds?.join(',') || ''
+                  )}`
+                )
+                  .then(res => res.json())
+                  .then(res => res as Response);
+              }
+              if (integration.type === 'wonder') {
+                let url = `https://external-api.wonderapp.co/v1/coordinape/contributions?org_id=${integration.data.organizationId}&epoch_start=${epoch.start_date}&epoch_end=${epoch.end_date}`;
+                if (integration.data.podIds) {
+                  for (const podId of integration.data.podIds) {
+                    url += `&pod_ids=${podId}`;
+                  }
+                }
+                return fetch(url)
+                  .then(res => res.json())
+                  .then(res => res as Response)
+                  .then(res => res);
```

Comment:
```suggestion
```

@terryli0095 do we need line 77? Seems it does't do anything...

Reply:
fixed